package.json

```diff
@@ -1,36 +1,24 @@
 {
-  "name": "create-dao",
-  "version": "1.4.3",
-  "private": "true",
-  "description": "",
+  "name": "root",
+  "version": "0.0.0",
+  "private": true,
   "scripts": {
     "dev": "yarn workspace @create-dao/next-app dev",
     "build": "yarn workspace @create-dao/next-app build",
     "start": "yarn workspace @create-dao/next-app start",
     "lint": "yarn workspace @create-dao/next-app lint",
     "chain": "yarn workspace @create-dao/next-app chain",
     "compile": "yarn workspace @create-dao/next-app compile",
-    "deploy": "yarn workspace @create-dao/next-app deploy"
+    "deploy": "yarn workspace @create-dao/next-app deploy",
+    "update-version": "lerna version --force-publish --no-private",
+    "publish-latest": "leran publish from-package",
+    "build:npx:create-dao": "node ./packages/create-dao/index.js"
```

Comment:
What does this command do? Does it execute the script that the users will execute? i.e. Can this command be used to test out any changes locally?

Reply:
The "deploy" script is to deploy contracts for the create-dao workspace
The "update-version" script will update the npm packages for publishing
The  "publish-latest" script will publish the packages
The "start:npx:create-dao" script (I just changed the name in latest update)  will run a test of create-dao locally

I just added docs to this PR, please let me know if the docs are clear, doc writing isn't a strength :)